stable/ksoc-plugins/Chart.yaml

```diff
@@ -16,11 +16,12 @@ annotations:
   artifacthub.io/category: security
   # Possible kind options are added, changed, deprecated, removed, fixed and security.
   artifacthub.io/changes: |
-    - kind: Removed
-      description: Removed comments in values yaml file to reduce confusion when pushing to the AWS Marketplace
+    - kind: Added
+      description: Added optional support for Custom Resource ingestion in `ksoc-watch`.
   artifacthub.io/containsSecurityUpdates: "false"
   artifacthub.io/links: |
     - name: source
       url: https://github.com/ksoclabs/ksoc-plugins-helm-chart
     - name: support
       url: https://github.com/ksoclabs/ksoc-plugins-helm-chart/issues
+  artifacthub.io/prerelease: "true"
```

Comment:
This should allow us to test before everyone updates https://artifacthub.io/docs/topics/annotations/helm/